### PR TITLE
fix: (REPLAT-9831) update article/section analytics

### PR DESCRIPTION
**Pull request by jmars**

### CODE REVIEW

File: packages/article-skeleton/src/tracking/article-tracking-context.js [hunk 1]
```diff
@@ -28,10 +28,13 @@ export default Component =>
...
         edition_type: editionType,
-        published_time: get(data, "publishedTime", ""),
+        published_time: published.toISODate(),
         past_edition_date:
-          editionType === "past 6 days" ? get(data, "publishedTime", "") : null,
-        parent_site: get(data, "publicationName", ""),
+          editionType === "past 6 days" ? published.toISODate() : null,
+        parent_site:
+          get(data, "publicationName", "") === "TIMES"
+            ? "the times"
```
Comment (aronshamash) [inline packages/article-skeleton/src/tracking/article-tracking-context.js]: do we not have this mapping of `publicationName` to analytics publication name somewhere already ?

**Comment by aronshamash** [comment]
@jmars needs desc and JIRA link

File: packages/article-skeleton/src/tracking/article-tracking-context.js (hunk 1 above)
Comment (apostolnikov) [inline packages/article-skeleton/src/tracking/article-tracking-context.js]: Is this trenary applicable to the web as well? What's the reason we want the parent_site to be lower case?

**Comment by ValGeorgiev** [comment]
This needs to be confirmed if its needed or no

**Comment by ValGeorgiev** [comment]
Not needed

**Pull request closed without merging**